lib/attributes/backend/dpcpp/kernel.cpp

```diff
@@ -19,15 +19,15 @@ using namespace oklt;
 using namespace clang;
 
 const std::string EXTERN_C = "extern \"C\"";
-const std::string DPCPP_ADDITIONAL_ARGUMENTS = "sycl::queue * queue_,sycl::nd_range<3> * range_";
+// const std::string DPCPP_ADDITIONAL_ARGUMENTS = "sycl::queue * queue_,sycl::nd_range<3> * range_";
```

Comment:
Does it make sense to leave this commented line?

Reply:
It was dead code anyway. So I removed it.